test/Microsoft.DotNet.Tools.Publish.Tests/Microsoft.DotNet.Tools.Publish.Tests.cs

```diff
@@ -31,13 +35,20 @@ public static IEnumerable<object[]> PublishOptions
             }
         }
 
+        public PublishTests() : base()
+        {
+            RepoRoot = Path.Combine(Directory.GetCurrentDirectory(), "..\\..\\");
```

Comment:
Agreed this isn't great.
We needed the global.json before when we were restoring but it was set up incorrectly.
If we can rely on some environment variable (RepoRoot maybe) we could avoid this ugliness.

I'm not entirely convinced that we need it now that we're not restoring actually. I'll try removing it altogether and see what happens.